inference/normalize_regex_inference.py

```diff
@@ -4,7 +4,7 @@
 import sys
 from tqdm import tqdm
 from joblib import Parallel, delayed
-from inference.indic_num_map import INDIC_NUM_MAP
+from ctranslate2_inference.indic_num_map import INDIC_NUM_MAP
```

Comment:
This should not be imported ctranslate2_inference as the folder is deleted.

Reply:
Oops sorry, missed it. Fixed now, please check.